### PR TITLE
Make analysis use the ccc selected

### PR DESCRIPTION
**Pull request by local-minimum**
Fixes bug that caused ccc not to be respected.

Made selenium test for starting an analysis to verify that it actually does this.

### CODE REVIEW

File: scanomatic/ui_server/analysis_api.py [hunk 1]
```diff
@@ -104,15 +110,23 @@ def get_analysis_instructions(project=None):
             ["urls", "compile_instructions", "analysis_logs"],
             dict(
                 instructions={
-                    'grayscale': "one-time" if model.one_time_grayscale else "dynamic",
-                    'positioning': "one-time" if model.one_time_positioning else "dynamic",
+                    'grayscale':
+                    "one-time" if model.one_time_grayscale else "dynamic",
+                    'positioning':
+                    "one-time" if model.one_time_positioning else "dynamic",
```
Comment (gdetrez) [inline scanomatic/ui_server/analysis_api.py]: This indentation is a bit weird...

File: scanomatic/ui_server/analysis_api.py [hunk 2]
```diff
@@ -104,15 +110,23 @@ def get_analysis_instructions(project=None):
...
                     'compilation': model.compilation,
                     'compile_instructions': model.compile_instructions,
                     'email': model.email,
                     'pinning_matrices': model.pinning_matrices,
-                    'grid_model': {'gridding_offsets': model.grid_model.gridding_offsets,
-                                   'reference_grid_folder': model.grid_model.reference_grid_folder},
+                    'grid_model': {
+                        'gridding_offsets': model.grid_model.gridding_offsets,
+                        'reference_grid_folder':
+                        model.grid_model.reference_grid_folder},
```
Comment (gdetrez) [inline scanomatic/ui_server/analysis_api.py]: Likewise

File: tests/system/test_analysis.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    if payload.get('instructions'):
+        assert False, (
+            "Test environment is not clean. There's stuff in the output folder"
+            .format(payload)
+        )
+
+    browser.get(scanomatic + '/analysis')
+
+    elem = browser.find_element_by_id('compilation')
+    elem.send_keys('')
```
Comment (gdetrez) [inline tests/system/test_analysis.py]: What's the point of this?

Comment (local-minimum) [reply tests/system/test_analysis.py]: When it gets focus, it puts the `root/`-thing in the input and googling on how to give focus that was what I found.

File: scanomatic/ui_server/analysis_api.py (hunk 1 above)
Comment (local-minimum) [reply scanomatic/ui_server/analysis_api.py]: fixed

File: scanomatic/ui_server/analysis_api.py (hunk 2 above)
Comment (local-minimum) [reply scanomatic/ui_server/analysis_api.py]: fixed

Comment (gdetrez) [reply scanomatic/ui_server/analysis_api.py]: Doesn't look different :confused:

**Pull request merged**